course_catalog/api_test.py

```diff
@@ -13,7 +13,7 @@
 from course_catalog.factories import CourseFactory
```

Comment:
github won't let me comment on the relevant line, but the docstring up at the top of this file says 'Test course_catalog task_helpers'